pkg/channel/consolidated/status/status.go

```diff
@@ -0,0 +1,469 @@
+/*
+Copyright 2019 The Knative Authors.
```

Comment:
update your IDE template :-)

Reply:
I now see why that is :-) 


Once this is merged, let's have a GH issue that we, eventually, remove all this and go w/ the `knative/pkg` bits